tools/test-proxy/docker/dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0-focal AS build
+
+ENV \
+    NO_AT_BRIDGE=1 \
+    CERT_FOLDER=/var/certwork \
+    CERT_IMPORT_SH=apply-dev-cert.sh \
+    ASPNETCORE_ENVIRONMENT=Development
+
+# Install openssl
+RUN apt-get update \
+    && apt-get install -y \
+        libsecret-1-dev \
+        libssl-dev
+
+# prep the machine dev certificate
+RUN mkdir $CERT_FOLDER
+
+ADD dev_certificate/$CERT_IMPORT_SH $CERT_FOLDER
+ADD dev_certificate/dotnet-devcert.pfx $CERT_FOLDER
+ADD dev_certificate/dotnet-devcert.crt $CERT_FOLDER
+
+RUN sed -i -e 's/\r$//' $CERT_FOLDER/$CERT_IMPORT_SH
+
+RUN chmod +x $CERT_FOLDER/$CERT_IMPORT_SH \
+    && $CERT_FOLDER/$CERT_IMPORT_SH \
+    && rm $CERT_FOLDER/$CERT_IMPORT_SH
+
+# install the package
+RUN dotnet tool install azure.sdk.tools.testproxy \
```

Comment:
I think we could improve the dockerfile to use a multi-stage build to significantly reduce the image size.  But we can do this in a later PR.